cmd/main.go

```diff
@@ -246,40 +225,27 @@ func main() {
 		lead = leader.NewElector(k8sClient, Options.LeaderConfig, "assisted-service-leader-election-helper",
 			log.WithField("pkg", "monitor-runner"))
 
-		err = lead.StartLeaderElection(context.Background())
-		if err != nil {
-			log.WithError(err).Fatalf("Failed to start leader")
-		}
+		mustSucceedf(lead.StartLeaderElection(context.Background()), "Failed to start leader")
 
 		ocpClient, err = k8sclient.NewK8SClient("", log)
-		if err != nil {
-			log.WithError(err).Fatalf("Failed to create client for OCP")
-		}
+		mustSucceedf(err, "Failed to create client for OCP")
 
 	case deployment_type_onprem, deployment_type_ocp:
 		lead = &leader.DummyElector{}
 		autoMigrationLeader = lead
 		// in on-prem mode, setup file system s3 driver and use localjob implementation
 		objectHandler = s3wrapper.NewFSClient("/data", log)
-		if objectHandler == nil {
```

Comment:
Was it removed on purpose?

Reply:
I looked at the function and it return a status object without any logic.